src/_provider/MyAnimeList_api/single.ts

```diff
@@ -72,6 +74,37 @@ export class Single extends SingleAbstract {
     }
   }
 
+  _getStartDate() {
+    return this.animeInfo.my_list_status.start_date;
```

Comment:
How do you suggest these situations to be handled? Ideally it is always converted to `YYYY-MM-DD` format in `getStartDate()` and `getFinishDate()` so that other sites can parse it. Perhaps it could round down (`2022` -> `2022-01-01`, `2018-05` -> `2018-05-01`, etc)?

Reply:
Yeah I think that is the better way. Alternative would be returning null in theses cases, but that is not nice either.

I think rounding and returning null for the impossible combinations like year and day, in case they really exist. Is the way to go.

Reply:
Fixed